### PR TITLE
[Docs](bangc-ops): fix mlu_op.h

### PR DESCRIPTION
**Pull request by tudejiang79**
Thanks for your contribution and we appreciate it a lot. 

## 1. Motivation

规范 mlu-ops仓 r0.4分支的mlu_op.h接口注释

## 2. Modification

1）bangc-ops/mlu_op.h

### CODE REVIEW

**Review by WangYuan-cn — APPROVED** [review]

**Review by DanieeelLiu — APPROVED** [review]

File: bangc-ops/mlu_op.h [hunk 1]
```diff
@@ -761,149 +952,205 @@ mluOpSetTensorDescriptorEx(mluOpTensorDescriptor_t desc,
...
 mluOpStatus_t MLUOP_WIN_API
 mluOpSetTensorDescriptorOnchipDataType(mluOpTensorDescriptor_t desc, mluOpDataType_t onchip_dtype);
 
 // Group:Tensor
 /*!
- *  @brief Sets the \b position factor to the descriptor \b desc of fixed-point data in
- *  fixed-point quantization. 
+ * @brief Sets the \b position factor to the descriptor \b desc of fixed-point data in
+ * fixed-point quantization. It is used in ::MLUOP_QUANTIZE_POSITION mode.
  *
```
Comment (GraceGuanH) [inline bangc-ops/mlu_op.h:1045]: It is used in ::MLUOP_QUANTIZE_POSITION mode  删除这句话，头文件中没有MLUOP_QUANTIZE_POSITION

File: bangc-ops/mlu_op.h [hunk 2]
```diff
@@ -761,149 +952,205 @@ mluOpSetTensorDescriptorEx(mluOpTensorDescriptor_t desc,
...
  */
 mluOpStatus_t MLUOP_WIN_API
 mluOpSetTensorDescriptorPosition(mluOpTensorDescriptor_t desc, int position);
 
 // Group:Tensor
 /*!
- *  @brief Sets the \b position and \b scale factors to the descriptor of fixed-point data in
- *  fixed-point quantization. 
+ * @brief Sets the \b position and \b scale factors to the descriptor of fixed-point
+ * data in fixed-point quantization. It is used in ::MLUOP_QUANTIZE_POSITION_SCALE mode.
```
Comment (GraceGuanH) [inline bangc-ops/mlu_op.h:1081]: It is used in ::MLUOP_QUANTIZE_POSITION_SCALE mode.  删除

File: bangc-ops/mlu_op.h [hunk 3]
```diff
@@ -761,149 +952,205 @@ mluOpSetTensorDescriptorEx(mluOpTensorDescriptor_t desc,
...
  */
 mluOpStatus_t MLUOP_WIN_API
 mluOpSetTensorDescriptorPositionAndScale(mluOpTensorDescriptor_t desc, int position, float scale);
+
 // Group:Tensor
 /*!
- *  @brief Sets the \b position, \b scale and \b offset factors to the descriptor of fixed-point
- *  data in fixed-point quantization. 
+ * @brief Sets the \b position, \b scale and \b offset factors to the descriptor of fixed-point
+ * data in fixed-point quantization. It is used in ::MLUOP_QUANTIZE_POSITION_SCALE_OFFSET mode.
```
Comment (GraceGuanH) [inline bangc-ops/mlu_op.h:1120]: It is used in ::MLUOP_QUANTIZE_POSITION_SCALE_OFFSET mode.  删除

File: bangc-ops/mlu_op.h (hunk 1 above)
Comment (tudejiang79) [reply bangc-ops/mlu_op.h:1045]: 已修改

File: bangc-ops/mlu_op.h [hunk 4]
```diff
@@ -1847,73 +2351,72 @@ mluOpGetPolyNmsWorkspaceSize(mluOpHandle_t handle,
...
- *  The size of extra space.
- *  @param[in] output_desc
- *  The descriptor of the output tensor. For detailed information,
- *  see ::mluOpTensorDescriptor_t.
- *  @param[out] output
- *  Pointer to the MLU memory that stores the output tensor.
- *  @param[in] output_size
- *  Pointer to the MLU memory that stores the output tensor. Indicates
- *  the number of return values of output.
+ * @brief Polygon Non Maximum Suppression.
```
Comment (GraceGuanH) [inline bangc-ops/mlu_op.h:2354]: 这是新加的接口吗？

File: bangc-ops/mlu_op.h (hunk 2 above)
Comment (tudejiang79) [reply bangc-ops/mlu_op.h:1081]: 已修改

File: bangc-ops/mlu_op.h (hunk 3 above)
Comment (tudejiang79) [reply bangc-ops/mlu_op.h:1120]: 已修改

**Review by GraceGuanH — COMMENTED** [review]
Please check the comments.

File: bangc-ops/mlu_op.h (hunk 4 above)
Comment (GraceGuanH) [reply bangc-ops/mlu_op.h:2354]: 先合入吧，我和算子owner再确认

**Review by GraceGuanH — APPROVED** [review]

**Pull request merged**